integration_tests/test_api.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+go install card-match/cmd/api
+
+$(api --port 8080) &
```

Comment:
@abatilo I'm worried that this could be a source of flake if the server starts to take longer to start up before I hit the endpoint.

Reply:
If you're really worried about it, throw in something like an until loop:
https://linuxize.com/post/bash-until-loop/

Or a for loop with retries:
https://github.com/abatilo/streamlit-preview-environments-demo/blob/master/.github/workflows/pr_opened.yml#L28